databases/zebra/Portfile

```
@@ -16,17 +17,19 @@ long_description    Zebra is a high-performance, general-purpose \
                     through exact boolean search expressions \
                     and relevance-ranked free-text queries.
 
-homepage            https://www.indexdata.com/resources/software/zebra/
+homepage            https://www.indexdata.com/resources/software/zebra
```

The canonical URL at this server does have the trailing slash so this change should be reverted.
```suggestion
homepage            https://www.indexdata.com/resources/software/zebra/
```